### ISSUE TITLE
Update Accessibility note on button cursors on hover

### ISSUE THREAD

**Issue by coliff** [trimble-oss/website-modus.trimble.com#345]
Feedback on Google Chat:
![image](https://user-images.githubusercontent.com/1212885/208419491-a1da83ff-ff55-40e9-b9b3-6beaa3e985fb.png)

I believe the style guide a11y note to be incorrect, buttons should have `cursor: pointer` (hand) to show they are enabled/active, just as links, checkboxes, switches are. Disabled buttons use default mouse pointer cursor so it's important that non-disabled buttons are differentiated.

Side note: Bootstrap itself uses `cursor: pointer` - PR for adding it with some reasoning is explained at the link `https://github.com/twbs/bootstrap/pull/25082`

What do you think @enowak1031 ?

**Comment by enowak1031** [comment]
I think you might be right. It should be a hand.
